### ISSUE TITLE
Shared framework ships runtimeconfig.json with unwanted policy settings

### ISSUE THREAD

**Issue by GrabYourPitchforks** [dotnet/sdk#32969]
See https://github.com/dotnet/sdk/issues/32941#issuecomment-1571102871 for some additional context. The shared framework's runtimeconfig.json file contains this:

```json
{
  "runtimeOptions": {
    "tfm": "net8.0",
    "configProperties": {
      "System.Reflection.Metadata.MetadataUpdater.IsSupported": false,
      "System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization": false
    }
  }
}
```

The _configProperties_ element is intended to be app-specific policy and shouldn't be included as part of the shared framework install. It looks like what's happening is that when we build the shared framework and publish it, we're also including its _runtimeconfig.json_ file as an asset to be redistributed without first stripping the app-specific policy sections from the file.

If this issue is fixed in the .NET 8 timeframe, please revert https://github.com/dotnet/sdk/pull/33004.

**Comment by marcpopMSFT** [comment]
@GrabYourPitchforks should we keep this open to revert your workaround?

**Comment by GrabYourPitchforks** [comment]
@marcpopMSFT Yes. The folks I spoke with agreed that this issue represents a legitimate bug in how the shared framework is built & redistributed. Once this bug is fixed, the workaround I checked in earlier should be reverted.

**Comment by marcpopMSFT** [comment]
@GrabYourPitchforks is there a pointer to the PR/issue tracking the bug in the shared framework so we know when to revert the workaround?

**Comment by elinor-fung** [comment]
I believe @vitek-karas fixed it on the arcade side with https://github.com/dotnet/arcade/pull/13844. That flowed into runtime with https://github.com/dotnet/runtime/pull/87996.

**Comment by vitek-karas** [comment]
This has been fixed for runtime and WindowsDesktop frameworks. Unfortunately ASP.NET doesn't use Arcade to build its shared framework package and has the same bug: https://github.com/dotnet/aspnetcore/issues/49486
(Note that ASP.NET makes it worse due to the build environment they run it in).

**Comment by marcpopMSFT** [comment]
Thanks for the update. Sounds like I should wait for aspnet to resolve this and then I can close this issue.

**Comment by marcpopMSFT** [comment]
Old issue triage: Closing as the aspnet item was resolved.
